README.md

```diff
@@ -1,2 +1,18 @@
-# pipeline-library
-A repository of Jenkins pipeline files we can reference from elsewhere
+# What This Repo Is
+
+A repository of application centric [Jenkins pipeline files](https://jenkins.io/doc/book/pipeline/) that be references or reused for appropriate applications.
```

Comment:
`s/that be references/that can be referenced/`

Reply:
Fixed